e2e-tests/run-tests.sh

```diff
@@ -51,6 +51,7 @@ else
   export ODK_URL="${ODK_PROTOCOL}${ODK_DOMAIN}:$ODK_PORT"
 fi
 
+export PW_EXPERIMENTAL_SERVICE_WORKER_NETWORK_EVENTS=1
```

Comment:
Playwright doesn't interfere with the working of service worker by default. 

https://playwright.dev/docs/service-workers-experimental